.github/workflows/build_linux.yml

```diff
@@ -0,0 +1,40 @@
+name: linux-wheels
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get tags
+      shell: bash
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
+    - uses: ./.github/workflows/actions/manylinux2010_x86_64
+    - name: copy manylinux wheels
+      run: |
+        mkdir dist
+        cp wheelhouse/cf_units*-manylinux2010_x86_64.whl dist/
+
+    - name: CheckFiles
+      shell: bash
+      run: |
+        ls dist
+    
+    - name: Test wheels
+      shell: bash
+      run: |
+        python -m pip install wheel twine pytest
+        cd dist && python -m pip install cf_units*.whl && python -m pytest --pyargs cf_units
+        python -m twine check *
+
+    - name: Publish a Python distribution to PyPI
+      uses: pypa/gh-action-pypi-publish@master
```

Comment:
We should pin this to a specific stable release version, say `@release/v1.4.2` which is the latest.

But I'll do that as a separate PR and ping you on that... I think that you've been though enough already

![pypi-publich-github-action](https://user-images.githubusercontent.com/2051656/127572940-c8c2cc36-7688-45b6-ba87-ad7e97dfffa6.png)